src/components/support/controllers.tsx

```diff
@@ -309,6 +309,28 @@ function validateInviteUsers({ invite_users }: ISignupForm): ReadonlyArray<IVali
   return errors;
 }
 
+function validateAdditionalUserYesButEmpty({ invite_users, additional_users }: ISignupForm): ReadonlyArray<IValidationError> {
+  const errors = [];
+  let userFieldsEmpty = true;
+  if (invite_users === 'yes' && additional_users) {
+    /* eslint-disable @typescript-eslint/prefer-for-of */
+    for (let i = 0; i < additional_users.length; i++) {
+      if (additional_users[i].email) {
+        userFieldsEmpty = false
+      }
+    }
+    /* eslint-enable @typescript-eslint/prefer-for-of */
+    if (userFieldsEmpty) {
+      errors.push({
+        field: 'additional_users-0',
+        message: 'Enter at least one additional user email address',
+      });
+    }
+  }
+
+  return errors;
+}
```

Comment:
I think this function could be somewhat simplified.

```typescript
function validateAdditionalUserYesButEmpty({ invite_users, additional_users }: ISignupForm): ReadonlyArray<IValidationError> {
  const errors = [];

  if (additional_users.some(user => user.email !== undefined)) {
    errors.push({
        field: 'additional_users-0',
        message: 'Enter at least one additional user email address',
      });
  }

  return errors;
}
```

What do you think?

Reply:
🏅 forgot about `.some`

Reply:
hmm, using some will still return this message when something is entered in field 2 or 3